src/MessagePack.UnityClient/Assets/Scripts/MessagePack/Formatters/StandardClassLibraryFormatter.cs

```diff
@@ -465,6 +465,27 @@ private BigIntegerFormatter()
 
         public void Serialize(ref MessagePackWriter writer, System.Numerics.BigInteger value, MessagePackSerializerOptions options)
         {
+#if NETCOREAPP2_1
+            if (!writer.OldSpec)
```

Comment:
Is it worth supporting OldSpec by using Str8 in that case?

Reply:
Writing headers outside of Writer is sensitive, so I wanted to avoid branching on OldSpec, which would be rarely used.
Also, given that OldSpec is mainly for compatibility (with other languages, with old libraries), I don't think it needs to be supported.